### PR TITLE
added names, fastpath parameters explanation to pandas.Series

### PR DESCRIPTION
**Pull request by rmunjal29**
- [x] closes #xxxx
- [x] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [x] whatsnew entry

### CODE REVIEW

**Comment by jbrockmendel** [comment]
Can you use a more informative title?

Looks like a .ipynb file got included accidentally?

**Comment by rmunjal29** [comment]
Yes sure I will do that.

And could you please ignore .ipynb file. I accidently got uploaded.

**Comment by jbrockmendel** [comment]
Thanks.  We can ignore the extra file for now, but you'll need to remove it before this can be merged.

Looks like the lint checker is unhappy about the removed newline.  Can you run `black pandas/core/series.py`

**Review by WillAyd — CHANGES_REQUESTED** [review]
Minor comments. If you can address and remove the ipynb file should be good to go

File: Untitled.ipynb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
```
Comment (jreback) [inline Untitled.ipynb]: need to revert this file

**Review by jreback — CHANGES_REQUESTED** [review]
also merge master

**Comment by WillAyd** [comment]
@rmunjal29 can you address comments?

**Comment by rmunjal29** [comment]
Already updated the changes

Regards,
Ritika

**Comment by WillAyd** [comment]
Looks like there is a notebook in the diff - can you remove that?

**Review by WillAyd — APPROVED** [review]

**Comment by WillAyd** [comment]
Thanks @rmunjal29

**Pull request merged**